wc_mugdha/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var (
+	lineCount int
+	charCount int
+	wordCount int
+	wordEnd   uint8
+)
+
+func main() {
+	allArgs := os.Args[1:]
+
+	for _, fileName := range allArgs {
+		lineCount, wordCount, charCount = 0, 0, 0
+		f, err := os.Open(fileName)
+		if err != nil {
+			erroredExit(err)
+		}
+		defer f.Close()
+
+		bytes := make([]byte, 64)
```

Comment:
No magic numbers. There's a helpful linter - https://github.com/tommy-muehle/go-mnd

```go
const maxBufferSize = 64
```

```suggestion
		bytes := make([]byte, maxBufferSize)
```